cmd/file-server/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
```

Comment:
We use `github.com/sirupsen/logrus` for logs. I'm not sure if that log there will show up.